src/controllers/dashboard/devices/devices.js

```diff
@@ -1,10 +1,40 @@
 define(['loading', 'dom', 'libraryMenu', 'globalize', 'scripts/imagehelper', 'date-fns', 'dfnshelper', 'emby-button', 'emby-itemscontainer', 'cardStyle'], function (loading, dom, libraryMenu, globalize, imageHelper, datefns, dfnshelper) {
     'use strict';
+    // Local cache of loaded
+    let deviceIds = [];
 
     function canDelete(deviceId) {
         return deviceId !== ApiClient.deviceId();
     }
 
+    function deleteViaApi(id) {
```

Comment:
I would much rather have this moved to the API client itself.

Reply:
Good call, will and link to this PR.

To be clear I'm assuming you mean adding it as a method off the `ApiClient` class [here](https://github.com/jellyfin/jellyfin-apiclient-javascript/blob/master/src/apiClient.js), right?

Reply:
Exactly :)

Reply:
Done as part of jellyfin/jellyfin-apiclient-javascript#59

After the PR is merged, I'll update this PR to use the method and latest apiclient version in `package.json`

Reply:
@masterkoppa the new methods should be available now if you'd like to update this pull request.

Reply:
@dkanada: Thanks for the reminder, I'll find some time soon to update this with that change.